firefox-ios/firefox-ios-tests/Tests/XCUITests/TodayWidgetTests.swift

```diff
@@ -160,7 +252,7 @@ class TodayWidgetTests: BaseTestCase {
 
     private func tapOnWidget(widgetType: String) {
         let widget = springboard.buttons.matching(NSPredicate(format: "label CONTAINS[c] %@", widgetType)).element
-        mozWaitElementHittable(element: widget, timeout: 5)
+        mozWaitElementHittable(element: widget, timeout: 10)
```

Comment:
IIRC we have variables for timeout, TIMEOUT and TIMEOUT_LONG..

Reply:
Replaced values with TIMEOUT variable